<launch>
<!-- 设置地图的配置文件 -->
<arg name="map" default="cloister_gmapping.yaml" />
<!-- cloister_gmapping racecar -->

    <!-- 运行地图服务器，并且加载设置的地图-->
    <node name="map_server" pkg="map_server" type="map_server"
                     args="$(find turtlebot_formation2)/maps/$(arg map)" output="screen"/>
	<param name="frame_id" value="tb3_0/map" />
	<remap from="/cmd_vel" to="tb3_0/cmd_vel"/>
	<remap from="/scan" to="tb3_0/scan"/>
	<remap from="tb3_0/odom" to="odom"/>

<!--node pkg="tf" type="static_transform_publisher" name="map_odom_broadcaster1" args="0 0 0 0 0 0 0 map odom 100" /-->
<!--node pkg="tf" type="static_transform_publisher" name="odom_base_broadcaster1" args="0 0 0 0 0 0 0 odom ares1/base_link 100" /-->

	
    <!-- 运行move_base节点 -->
    <include file="$(find turtlebot_formation2)/launch/include/move_base.launch"/>

    <!-- 启动AMCL节点 -->
    <include file="$(find turtlebot_formation2)/launch/include/Amcl_tb3_0.launch" />

    <!-- 运行rviz -->
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find turtlebot_formation2)/rviz/turtlebot3_navigation.rviz"/>

</launch>
